lua/clangd_extensions/switch_source_header.lua

```diff
@@ -1,6 +1,8 @@
 local api = vim.api
 local nvim_get_current_buf = api.nvim_get_current_buf
 
+---@param _err lsp.ResponseError
+---@param uri string
 local function handler(_err, uri)
     if not uri or uri == "" then
```

Comment:
can you change `_err` to `err` and line 7 to `if err or (not uri) or (uri == "")`? [this](https://clangd.llvm.org/extensions#switch-between-sourceheader) says it returns empty string when it can't find the alt file, [this](https://github.com/llvm/llvm-project/commit/111fe84bbc1b6530d02ed0f6547070eb37b32b8f) says it switched to `nil`, but i think we should still handle `err`

Reply:
`if err or (not uri) or (uri == "")` is pending